tools/AzDev/src/AzDev.csproj

```diff
@@ -1,14 +1,21 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <OutputType>library</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
Update to net8 cause (a) tooling doesn't need to be cross-platform (b) System.Reflection.MetadataLoadContext is unavailable on netstandard2.0